pkg/scheduler/api/types.go

```diff
@@ -23,6 +23,12 @@ const (
 	// Pending means the task is pending in the apiserver.
 	Pending TaskStatus = 1 << iota
 
+	// Borrowing means that the task is allocated borrowing resources
+	// that are currently occupied by tasks.
+	// Task T is allocated to Node N as an Borrowing task if,
+	// and only if, N.IdleResource < T.RequestResource <= N.AllocatableResource.
+	Borrowing
```

Comment:
will that "Borrowing" resources will be return?

Reply:
see, this is exactly why I didn't want to change to single word name. CONFUSION!!

Reply:
Single word is better to me, but `Borrowing` and the comments does not describe it well.